stubs/requests/requests/_internal_utils.pyi

```diff
@@ -0,0 +1,14 @@
+from re import Pattern
+
+_VALID_HEADER_NAME_RE_BYTE: Pattern[bytes]
+_VALID_HEADER_NAME_RE_STR: Pattern[str]
+_VALID_HEADER_VALUE_RE_BYTE: Pattern[bytes]
+_VALID_HEADER_VALUE_RE_STR: Pattern[str]
+
+HEADER_VALIDATORS = {
+    bytes: (_VALID_HEADER_NAME_RE_BYTE, _VALID_HEADER_VALUE_RE_BYTE),
+    str: (_VALID_HEADER_NAME_RE_STR, _VALID_HEADER_VALUE_RE_STR),
+}
```

Comment:
```suggestion
from typing import Any

HEADER_VALIDATORS: dict[type, tuple[Pattern[Any], Pattern[Any]]]
```

Reply:
Happy to make this change, but why use `Any` instead of a typed dict with more specific types?

Reply:
I already applied the change to fix the flake8/pytype CI failure. A `TypedDict` won't work here, as a `TypedDict` has to have `str` keys, and this dictionary doesn't.

Reply:
Ah, because they're `type`. Makes sense, thanks